e2e/tests/manage.feature

```diff
@@ -1,7 +1,16 @@
 Feature: Manage an Approved Premises
-        Scenario: Creating a booking
+        Background:
                 Given I am logged in
+
+        Scenario: Creating a booking
+                When I access the premises homepage
                 And I see a list of premises
                 And I choose a premises
                 And I create a booking
-                Then I should see a confirmation screen
+                Then I should see a confirmation screen for my booking
+
+        Scenario: Creating a lost bed
+                When I access the premises homepage
+                And I choose a premises
+                And I create a lost bed
+                Then I should see a notification that the lost bed has been created
```

Comment:
Is it worth also have a basic unhappy path scenario, e.g.

```
Scenario: Fail to create a lost bed
  Given I'm managing a premises
  And I attempt to create a lost bed without the necessary information
  Then I should see that the lost bed has not been created
  And I should see a list of the problems encountered creating the lost bed
```

This might be helpful, as validation, error messaging is a big area of coupling...